src/cpp/core/include/core/r_util/RTokenizer.hpp

```diff
@@ -511,6 +511,28 @@ inline bool canContinueStatement(const RToken& rToken)
            canOpenArgumentList(rToken);
 }
 
+inline bool isSymbolNamed(const RToken& rToken,
+                          const std::wstring& name)
+{
+   // For strings, check if the content within the quotes
+   // is equal to the name provided. TODO: handle escaped
```

Comment:
That note implies that we don't handle someone naming a symbol something like `"a\"function" <- function(x) { ... }`, which not a big deal IMO.